src/pyodide-py/pyodide/_core.py

```diff
@@ -0,0 +1,26 @@
+# type: ignore
+import platform
+
+if platform.system() == "Emscripten":
+    from _core import JsProxy, JsBoundMethod, JsException
```

Comment:
It should have pyodide in its import name. Otherwise it's not clear at all that `_core` has anything to do with pyodide. Maybe `_pyodide_core` ?

Reply:
Sounds good.